docs: Aligned network page with last modifications (#6096)
eclipse/kura: docs/gateway-configuration/network-configuration.md

Since the network configuration shown on the screen may not be synchronized with the current state of the system, it can be updated pressing the **Refresh** button. This can be used also to force the reload of specific parameters like the RSSI or dynamic IP addresses. The refresh procedure reads all the needed parameters from the system and can take several seconds before updating.

!!! tip

>     It is recommended that the **TCP/IP**, **IPv4** or **IPv6** tab is configured first since it defines how the interface is going to be used.

Reviewer: ```suggestion
    It is advisable to configure the **TCP/IP**, **IPv4**, or **IPv6** tab first, as this determines how the interface will function.
```

Possible rewrite

## TCP/IP Configuration

The **TCP/IP**, **IPv4** and **IPv6** tabs contain the following configuration parameters:

- **Status**
    - Disabled: disables the selected interface (i.e., administratively down).
    - Enabled for LAN: designates the interface for a local network. It can be set as a DHCP server for hosts on the local network and can serve as a default gateway for those hosts; however, it cannot be set as an actual gateway interface for this device. That is, packets must be routed from this interface to another interface that is configured as WAN. The interface is automatically brought up at boot.
    - Enabled for WAN: designates the interface as a gateway to an external network. The interface is automatically brought up at boot.

>     - Not Managed: the interface will be ignored by Kura (available only for **IPv4**).

Reviewer: Should we invert this to (not available in IPv6) as the Not managed is available in the old networking (TCP/IP tab) and new networking but IPv4 tab only?
